### PR TITLE
Map Update Bug Fixes

### PR DESCRIPTION
**Pull request by aumu1031**

### CODE REVIEW

**Comment by aumu1031** [comment]
- Fixed special missions moving onto village tiles
- Adjusted mission rep rewards for increased times (missions will be reworked accordingly)

File: ui_components/build/travel/Travel.js [hunk 1]
```diff
@@ -544,9 +544,7 @@ function TravelActions({
     onPointerUp: handlePointerUp,
     onPointerLeave: handlePointerUp,
     onContextMenu: e => {
-      if (e.nativeEvent.pointerType === "touch") {
-        e.preventDefault();
-      }
+      e.preventDefault();
```
Comment (levimeahan) [inline ui_components/build/travel/Travel.js:547]: Ah, why was this needed?

**Review by levimeahan — APPROVED** [review]
LGTM!

File: ui_components/build/travel/Travel.js (hunk 1 above)
Comment (aumu1031) [reply ui_components/build/travel/Travel.js:547]: People were reporting that when holding down movement keys on mobile the context menu is popping up after a few seconds. For some reason the pointerType === "touch" check isn't inclusive enough so I'm trying the nuclear option.
